styles.css

```diff
@@ -0,0 +1,785 @@
+  color: #f2f1f1;
+}
+
+.about.footer-content a {
+  color: #f2f1f1;
+}
+
+/* Intro Section Styles
+************************** */
+
+.intro-subheading {
+  color: #ec5242;
+  font-size: 6vw;
+}
+
+h1 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 9vw;
+  color: #ec5242;
+  margin: 0 0.5rem;
+  filter: opacity(0.9);
+}
+
+h1 + p,
+hr + p.about {
+  padding: 10px;
+}
+
+h1 + p.home {
+  background-color: #d3d3d3;
+  border: 2px solid white;
+}
+
+hr + p.about {
+  background-color: white;
+  border: 2px solid white;
+}
+
+p.time {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 5.5vw;
+}
+
+.intro p {
+  margin: 0.5rem;
+}
+
+.intro {
+  background-image: url('graphics/bg.png');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  min-height: 120vw;
+}
+
+.intro.about {
+  background-image: url('graphics/bg-about.png');
+}
+
+.intro-container {
+  width: 84%;
+  margin: 0 auto;
+}
+
+/* Goals Section Styles
+************************** */
+
+.goals {
+  background-image: url('graphics/bg-repeat.png');
+  background-repeat: repeat;
+  background-size: 5vw;
+  padding: 3vw;
+}
+
+.goals img {
+  width: 18vw;
+}
+
+.goals h2,
+.goals p {
+  color: #f2f1f1;
+}
+
+.goals h2 {
+  font-size: 6vw;
+  text-align: center;
+  margin: 2vw 0 0;
+}
+
+.goal {
+  background-color: hsla(0, 0%, 83%, 0.2);
+  display: flex;
+  align-items: center;
+  padding: 3vw;
+  margin: 15px 5vw;
+}
+
+.goal p {
+  width: 34vw;
+}
+
+.action {
+  color: #ec5242;
+  font-size: 5vw;
+  width: 20vw;
+  margin: 0 2.5vw;
+}
+
+.agenda {
+  width: 70vw;
+  margin: 8vw auto;
+}
+
+/* Speakers Section Styles
+************************** */
+
+.speaker {
+  display: flex;
+  margin-bottom: 6vw;
+  justify-content: space-between;
+}
+
+.speaker-content {
+  width: 60vw;
+}
+
+.speaker h3 {
+  margin: 0 0 2vw;
+  font-size: 4vw;
+}
+
+.speakers h2 {
+  font-size: 5vw;
+  text-align: center;
+  margin: 6vw 0 0;
+}
+
+.speaker-subheading {
+  margin: 0 0 4vw 0;
+  font-size: 3vw;
+  color: #ec5242;
+}
+
+.speaker-img-container {
+  position: relative;
+  width: 37vw;
+  height: 25vw;
+}
+
+.brain {
+  width: 15vw;
+  position: absolute;
+  top: 2vw;
+}
+
+.speaker-img {
+  width: 25vw;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.speaker p {
+  margin: 2vw 0;
+}
+
+.speakers > hr {
+  margin-bottom: 8vw;
+}
+
+.speaker hr {
+  border: 1px solid #272a31;
+  margin-left: 0;
+  width: 4vw;
+}
+
+/* Organizers Section Styles
+************************** */
+
+.organizers {
+  background-color: #272a31;
+  padding: 30px 30px 50px;
+}
+
+.organizers img {
+  width: 150px;
+}
+
+.organizers h2 {
+  color: #f2f1f1;
+  font-size: 5vw;
+  text-align: center;
+  margin: 0;
+}
+
+.organizers hr {
+  margin-bottom: 6vw;
+}
+
+.organizers ul {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 4vw;
+}
+
+/* Our Team Section Styles
+************************** */
+
+.team {
+  padding: 30px 30px 50px;
+}
+
```

Comment:
- [ ] Please, let's update the team section by adding more content following the design pattern according to the design as much as we can below:

**original section implementation pattern**
![hett](https://user-images.githubusercontent.com/59397159/170781839-a981ae68-63de-4075-abc3-85e69821e237.png)


**your design implementation pattern**
![designn](https://user-images.githubusercontent.com/59397159/170781841-ca898e12-6751-4b51-821e-4994c4ef80ce.png)